### PR TITLE
Add Axem

### PR DESCRIPTION
**Pull request by magyta93**
## Requirements

 * [ ] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [ ] Pricing information is clearly visible without signup or phone calls
 * [ ] The submission mentions what is free
 * [ ] The submission is not already present in the list
 * [ ] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -720,6 +720,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 
 ## Management System
 
+  * [Axem](https://axemsolutions.io/) — DevOps for Embedded. Revolutionizing the embedded development experience.
```
Comment (ripienaar) [inline README.md:723]: Please explain what SaaS services this provide and what is free about them.

Comment (magyta93) [reply README.md]: fixed it!

File: README.md [hunk 2]
```diff
@@ -720,7 +720,9 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 
 ## Management System
 
+  
   * [bitnami.com](https://bitnami.com/) — Deploy prepared apps on IaaS. Management of 1 AWS micro instance free
+  * [DEM](https://axemsolutions.io/dem_doc/index.html) — Development Environment Manager Open-source, free command line tool to manage containerized Development Environments
```
Comment (ripienaar) [inline README.md:725]: What SaaS features does this CLI expose and use?

(Please read the top of the README)

Comment (magyta93) [reply README.md:725]: Designed to work with containerized tools for embedded software development

Automatic Development Environment setup
Able to set up the exact same environment for each developer
Consistent Development Environment through the whole project life cycle

Comment (ripienaar) [reply README.md:725]: It's a local only tool right? No SaaS involvement?

Comment (magyta93) [reply README.md:725]: Now it's only local

Comment (ripienaar) [reply README.md:725]: Thanks. Feel free to come back when it’s actually a SaaS as the list requires.

**Pull request closed without merging**